mmdet/core/anchor/anchor_generator.py

```diff
@@ -230,6 +232,29 @@ def grid_anchors(self, featmap_sizes, device='cuda'):
             multi_level_anchors.append(anchors)
         return multi_level_anchors
 
+    def grid_anchors(self, featmap_size, stride=16, device='cuda'):
+
+        warnings.warn('``grid_anchors`` would be deprecated soon. Please use '
+                      '``grid_priors``')
+        return self.grid_priors(
+            self, featmap_size=featmap_size, stride=stride, device=device)
+
+    def sparse_priors(self, level_idx, featmap_size, dtype, device, topk_inds):
```

Comment:
dtype and device should be the last two arguments with default values.
Suggest the argument order as topkinds, featmap_size, level_idx, dtype, device based on their importance.